bindings/c#/run.me

```diff
@@ -25,27 +25,55 @@ using System.IO;
 namespace supranational { public static class blst {
 
 #if NET5_0_OR_GREATER
-private static string dll;
+
+private static readonly string dll;
 
 static blst()
 {
-    if (String.IsNullOrEmpty(dll)) {
-        var name = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "blst.dll"
-                 : RuntimeInformation.IsOSPlatform(OSPlatform.OSX)     ? "libblst.dll.dylib"
-                 : "libblst.dll.so";
-
-        var dir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-        var arch = RuntimeInformation.ProcessArchitecture switch {
-            Architecture.X64   => "x64",
+    if (String.IsNullOrEmpty(dll))
+    {
```

Comment:
Minimize the difference by moving the name back here and adhering to the original formatting.

Reply:
There is great value in making commits concentrate specifically on code modifications and not formatting. I've exercised my editorial power to distill this PR accordingly.

Reply:
understood. I think I undid most if not all of the formatting changes.